src/main/java/ch/njol/skript/expressions/ExprYawPitch.java

```diff
@@ -71,26 +84,60 @@ public Number convert(final Object object) {
 	@SuppressWarnings({"null"})
 	@Override
 	public Class<?>[] acceptChange(final ChangeMode mode) {
-		if (mode == ChangeMode.SET || mode == ChangeMode.ADD || mode == ChangeMode.REMOVE)
+		if (mode == ChangeMode.SET || mode == ChangeMode.ADD || mode == ChangeMode.REMOVE) {
+			if (Player.class.isAssignableFrom(getExpr().getReturnType()) && !PLAYER_SUPPORT) {
+				Skript.error("Changing a rotation of a player requires Paper 1.19 and above.");
+				return null;
+			}
 			return CollectionUtils.array(Number.class);
+		}
 		return null;
 	}
 
 	@SuppressWarnings("null")
 	@Override
-	public void change(Event e, Object[] delta, ChangeMode mode) {
+	public void change(Event event, Object[] delta, ChangeMode mode) {
 		if (delta == null)
 			return;
 		float value = ((Number) delta[0]).floatValue();
-		for (Object single : getExpr().getArray(e)) {
-			if (single instanceof Location) {
+		for (Object single : getExpr().getArray(event)) {
+			if (single instanceof Entity) {
+				changeEntity(((Entity) single), value, mode);
+			} else if (single instanceof Location) {
 				changeLocation(((Location) single), value, mode);
 			} else if (single instanceof Vector) {
 				changeVector(((Vector) single), value, mode);
 			}
 		}
 	}
 
+	private void changeEntity(Entity entity, float value, ChangeMode mode) {
+		if (entity instanceof Player && !PLAYER_SUPPORT)
+			return;
+		float yaw = entity.getLocation().getYaw();
+		float pitch = entity.getLocation().getPitch();
+		switch (mode) {
+			case SET:
+				if (usesYaw)
+					yaw = convertToPositive(value);
+				else
+					pitch = value;
```

Comment:
Chained if-else blocks should use braces.
Same for below
```suggestion
				if (usesYaw) {
					yaw = convertToPositive(value);
				} else {
					pitch = value;
				}
```